core/domain/wipeout_service.py

```diff
@@ -466,6 +490,24 @@ def delete_user(
     _delete_models(user_id, models.Names.LEARNER_GROUP)
 
 
+def _verify_profile_picture_is_deleted(username: str) -> None:
+    """Verify that the profile picture is deleted.
+
+    Args:
+        username: str. The username of the user.
+    """
+    if username is None:
+        return
```

Comment:
Why is this needed, per typing info `username` cannot be `None`.

Reply:
I have updated this.